docker-compose.yml

```diff
@@ -20,15 +20,15 @@ services:
       context: ./docker
       dockerfile: minio.Dockerfile
     environment:
-      INITIAL_BUCKETS: "local/exam-archive-local"
+      INITIAL_BUCKETS: 'local/exam-archive-local'
     volumes:
       - exam-archive-minio-data:/data
     ports:
       - 127.0.0.1:9001:9000
       - 127.0.0.1:9002:9001
 
   db:
-    image: postgres:11.16
+    image: postgres:15.0
```

Comment:
RDS instance is still 11 so might want to stay on the same version until we find the time to upgrade. Probably 11.17 or whatever is latest.

Reply:
changed to 11.7